src/main/java/com/nvidia/spark/rapids/jni/CastStrings.java

```diff
@@ -109,6 +109,21 @@ public static ColumnVector toIntegersWithBase(ColumnView cv, int base,
       type.getTypeId().getNativeId()));
   }
 
+  /**
+   * Converts an integer column to a strings column with either decimal or hexadecimal values 
+   * based on the base provided. The hexadecimal value will be returned with the leading
+   * zeros dropped
```

Comment:
Again maybe in a "Note:" say that there are no leading zeros or padding of digits to a particular bit width